views/js/controller/creator/views/property.js

```diff
@@ -133,22 +133,23 @@ function($, ui, DataBinder, templates){
 
             $view.on('validated.group', function(e, isValid){
                 let classErrorAll = $container.find('span.validate-error');
-                let classErrorVisibleFields = $('.section-props:visible').find('span.validate-error');
+                let $testSection = $('.tlb-button-on').parents('.section').attr('id');
+                let testSectionId = '#' + $testSection;
+                let $propsSectionError = $('#section-props-'+ $testSection).find('span.validate-error');
+                let $propsItemError = $('.itemref-props').find('span.validate-error');
+                let $deleteButtonInCurrentSection = $(testSectionId).find('[data-testid="remove-section"]');
```

Comment:
Had them separated at first because I meant to treat the behavior separately but then didn't do it in this function and forgot to group them back, all done now thank you!